priv/gettext/fr/LC_MESSAGES/default.po

```diff
@@ -272,22 +272,22 @@ msgstr "Autonomie"
 #, elixir-format
 #: lib/teslamate_web/live/settings_live/index.html.leex:148
 msgid "ideal"
-msgstr "Estimée"
+msgstr "standard (Tesla)"
```

Comment:
I have mixed feelings about translations whose meaning differs so much from the original. `ideal` is not the standard range for everyone, for example. 

These terms come from the Tesla API. It would be nice if the translations could reflect this. A tooltip would be a better option for explanations of this kind.